core-test/src/main/java/org/openstack4j/api/telemetry/CapabilitiesV3Test.java

```diff
@@ -0,0 +1,27 @@
+package org.openstack4j.api.telemetry;
+
+import org.openstack4j.api.AbstractTest;
+import org.openstack4j.model.telemetry.Capabilities;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+
+import static org.testng.Assert.assertNotNull;
+
+@Test(suiteName = "CapabilitiesV3 Tests")
+public class CapabilitiesV3Test extends AbstractTest {
+    private static final String JSON_CAPABILITIES = "/telemetry/capabilities.json";
+
+    @Override
+    protected Service service() {
+        return Service.TELEMETRY;
+    }
+
+    @Test
+    public void getSampleTest() throws IOException {
+        respondWith(JSON_CAPABILITIES);
+        Capabilities capabilities = osv3().telemetry().capabilities().get();
```

Comment:
Shouldn't this be like `.get(<id>)` ? Else maybe `.list()` is be more consistent, or?